task.sql

```diff
@@ -38,3 +38,16 @@ CREATE TABLE ProductInventory (
     PRIMARY KEY (ID)
 );
 --rollback DROP TABLE ProductInventory;
+
+--changeset mate.academy:5 labels:0.0.2
+CREATE TABLE Users (
+    ID INT AUTO_INCREMENT PRIMARY KEY,
+    FirstName VARCHAR(255),
+    LastName VARCHAR(255),
+    Email VARCHAR(255)
+);
+--rollback DROP TABLE Users;
+
+--changeset mate.academy:6 labels:0.0.3
+CREATE INDEX idx_email ON Users(Email);
+--rollback DROP INDEX idx_email ON Users;
```

Comment:
![image](https://github.com/user-attachments/assets/3d4dd408-0bb2-4874-9e0b-e74f5797e295)
Name of index should be `Email`

```suggestion
--changeset mate.academy:6 labels:0.0.3
CREATE INDEX Email ON Users(Email);
--rollback DROP INDEX idx_email ON Users;
```